Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: help test auto-style code-style code-lint type-check lint lint-fix clean
```

Comment:
you don't need a makefile, i suggest you use the poetry CLI commands to do all this stuff.